docs/faq.md

```diff
@@ -269,6 +270,12 @@ Yes! Please [submit an issue or open a pull request][pr-issue-question] if this
 
 Yes! Please [submit an issue or open a pull request][pr-issue-question] if this does not work.
 
+Use `require('!style-loader!css-loader!video.js/dist/video-js.css')` to inject video.js CSS.
```

Comment:
I would add "Be sure to" on the front of this sentence to make it a bit more readable.